dashboard/package.json

```diff
@@ -41,7 +41,8 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "doc":"jsdoc -c jsdoc.json"
```

Comment:
I like that you sorted the library dependencies. Should we sort the script names, too? 😆 

Also, you should include the space after `"doc": ` like all the other script targets, for consistency.

Reply:
I guess the scripts are also sorted ,in this case not alphabetically,but priority wise 😆 ,don't you think so?